Source/Rows/ButtonRow.swift

```diff
@@ -61,7 +61,7 @@ open class _ButtonRowOf<T: Equatable> : Row<ButtonCellOf<T>> {
     required public init(tag: String?) {
         super.init(tag: tag)
         displayValueFor = nil
-        cellStyle = .default
+        cellStyle = .value1
```

Comment:
The ButtonRow should have `.default` as its default cellStyle. It can always be changed in the row initializer:

```swift
ButtonRow() { row in
    row.cellStyle = .value1
}
```